controllers/ControllerAdmin.php

```diff
@@ -53,10 +47,11 @@ public function profile(){
 			$this->avatar();
 		}else{
 			echo $this->_twig->render('profile.twig', ['logged' => TRUE,
-			                                          'user'   => $key['pseudo'],
-			                                          'value'  => $key['email'],
+			                                           'user'   => $key['pseudo'],
+			                                           'value'  => $key['email'],
+			                                           'admin' => $key['admin'],
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$key['admin']'.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)